src/components/PhotoAnalysis/PhotoAnalysis.tsx

```diff
@@ -80,12 +83,15 @@ const Analysis: React.FC = () => {
             onChange={identifyImageLabels}
           ></input>
           <span className="file-cta">
-            <span className="file-label">Choose a file…</span>
+            <span className="file-label">
+              <FontAwesomeIcon icon={faCamera} fontSize="25" />
+            </span>
           </span>
         </label>
       </div>
       <br />
-      <img src={imageSrc} alt="user's uploaded file" />
+
+      <img src={imageSrc} />
```

Comment:
If I merge my changes in before this CR is complete, it would be worth it to rebase first and then update this PR